src/main/scala/fr/acinq/bitcoin/Protocol.scala

```diff
@@ -288,6 +288,7 @@ object Message extends BtcSerializer[Message] {
   val MagicMain = 0xD9B4BEF9L
   val MagicTestNet = 0xDAB5BFFAL
   val MagicTestnet3 = 0x0709110BL
+  val MagicSigNet = 0x40CF030AL
```

Comment:
Can you link me to where that magic value comes from?

Reply:
from rust-bitcoin
https://github.com/rust-bitcoin/rust-bitcoin/blob/0.27.0/src/network/constants.rs#L93

Reply:
Thanks.